src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
+authors:
+  - katiehempenius
+date: 2019-07-29
+hero: hero.jpg
+alt: A pile of photos.
+description: |
+  Instructions on how to install Thumbor is an open-source image CDN and can be used for free to resize, compress, and transform images.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+  - images
+---
+
+Image CDNs make it easy to dynamically optimize the aesthetics and performance of your images. Unlike most image CDNs, Thumbor is  open-source and can be used for free to resize, compress, and transform images. It's suitable for production use; [Wikipedia](https://wikitech.wikimedia.org/wiki/Thumbor) and [Square](https://medium.com/square-corner-blog/dynamic-images-with-thumbor-a430a1cfcd87) both use Thumbor.
```

Comment:
And link to Thumbor's site the first time that you mention it: http://thumbor.org/